daemon/Makefile

```diff
@@ -13,7 +13,9 @@ $(TARGET): $(SOURCES) check-bindata
 	@$(ECHO_GO)
 	$(GO) build -i $(GOBUILD) -o $(TARGET)
 
-GO_BINDATA := $(QUIET) go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
+GO_BINDATA_BIN := $(GO) run ../vendor/github.com/cilium/go-bindata/go-bindata/*.go
```

Comment:
Could we just go install go-bindata? it's a little weird to me to invoke it so directly. The path includes the correct bin directory, right?

Reply:
it's `$(GO) run` compiles the program at runtime.

I don't like it either but I made this commit before adding the target `install-cilium-golang-deps`

Reply:
Assuming you agree, we could use the `-o` option to go build to place the tool binaries in a local `bin` directory, then modify path to prefer this `bin`. This also seems bad, though, since I usually expect make to use the PATH I give it...

All of this is to avoid asking the user to install the tools, right? I guess it also ensures the correct version is used.